src/libraries/Common/src/System/Runtime/InteropServices/ComEventsMethod.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Why is this necessary?  Is this file built into some project other than Corelib and that hasn't been annotated yet for nullable reference types? (Same question for the other occurrences in other files.)

Reply:
`Microsoft.CSharp` has not been